Nightly docs build fails due to missing imports

## 📝 Provide detailed reproduction steps (if any)

CI reports 329 errors due to importing files that are not available on npm.

```
  You can mark the path "@ckeditor/ckeditor5-image/docs/assets/img/icons/right.svg" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.

✘ [ERROR] Could not resolve "@ckeditor/ckeditor5-image/docs/assets/img/icons/side.svg"

    node_modules/umberto/source/ckeditor5/******/_snippets/features/image-style-custom.js:13:21:
      13 │ ...Icon from '@ckeditor/ckeditor5-image/docs/assets/img/icons/side...
         ╵              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  The path "./docs/assets/img/icons/side.svg" is not exported by package "@ckeditor/ckeditor5-image":

    node_modules/@ckeditor/ckeditor5-image/package.json:49:13:
      49 │   "exports": {
         ╵              ^

  You can mark the path "@ckeditor/ckeditor5-image/docs/assets/img/icons/side.svg" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.

✘ [ERROR] Could not resolve "@ckeditor/ckeditor5-ckbox/tests/_utils/ckbox-config.js"

    node_modules/umberto/source/ckeditor5/******/_snippets/features/image-style-custom.js:15:26:
      15 │ ...RL } from '@ckeditor/ckeditor5-ckbox/tests/_utils/ckbox-config....
         ╵              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  The path "./tests/_utils/ckbox-config.js" is not exported by package "@ckeditor/ckeditor5-ckbox":

    node_modules/@ckeditor/ckeditor5-ckbox/package.json:48:13:
      48 │   "exports": {
         ╵              ^

  You can mark the path "@ckeditor/ckeditor5-ckbox/tests/_utils/ckbox-config.js" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.

Error: Build failed with 329 errors:
```

The previous snippet adapter defined aliases to those files:

https://github.com/ckeditor/ckeditor5/blob/f90f949eb05971d63b26e66f1b482833afabd85c/scripts/docs/snippetadapter.mjs#L42-L61

Let's find a solution that fixes the snippets and doesn't force us to publish these files.

--- a/scripts/docs/snippetadapter.mjs
+++ b/scripts/docs/snippetadapter.mjs
@@ -11,9 +11,30 @@ import { createRequire } from 'module';
 import upath from 'upath';
 import { glob } from 'glob';
 import { build as esbuild } from 'esbuild';
+import { CKEDITOR5_ROOT_PATH } from '../constants.mjs';
 
 const require = createRequire( import.meta.url );
 const __dirname = upath.dirname( fileURLToPath( import.meta.url ) );
+
+/**
+ * Production CKEditor 5 documentation is built using packages published to npm. However, some of
+ * the imports used in snippets rely on code not available there. This map is used to resolve such
+ * imports to the actual source files not present in the "exports" field of the published packages.
+ */
+const RESOLVE_ALIAS_MAP = {
+	'@ckeditor/ckeditor5-image/docs/assets': upath.join(
+		CKEDITOR5_ROOT_PATH, 'packages', 'ckeditor5-image', 'docs', 'assets'
+	),
+	'@ckeditor/ckeditor5-core/tests/_utils/articlepluginset.js': upath.join(
+		CKEDITOR5_ROOT_PATH, 'packages', 'ckeditor5-core', 'tests', '_utils', 'articlepluginset.js'
+	),
+	'@ckeditor/ckeditor5-cloud-services/tests/_utils/cloud-services-config.js': upath.join(
+		CKEDITOR5_ROOT_PATH, 'packages', 'ckeditor5-cloud-services', 'tests', '_utils', 'cloud-services-config.js'
+	),
+	'@ckeditor/ckeditor5-ckbox/tests/_utils/ckbox-config.js': upath.join(
+		CKEDITOR5_ROOT_PATH, 'packages', 'ckeditor5-ckbox', 'tests', '_utils', 'ckbox-config.js'
+	)
+};
 
 /**
  * @param {Set<Snippet>} snippets Snippet collection extracted from documentation files.
@@ -112,6 +133,7 @@ async function buildSnippets( inputPath, outputPath, constants, imports ) {
 		format: 'esm',
 		target: 'es2022',
 		tsconfigRaw: {},
+		alias: RESOLVE_ALIAS_MAP,
 		loader: {
 			'.js': 'jsx',
 			'.svg': 'text'

